src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -368,7 +368,7 @@ export default class OrgPerson extends Mixins(EntityFilterMixin, CommonMixin) {
     }
   }
 
-  private isRoleLocked (role: string) : boolean {
+  private isRoleLocked (role: Roles) : boolean {
```

Comment:
Convention is that a singular would be called "Role" and a list or array of them would be called "Roles".
Small change, sorry, but what would you call an array of Roles? Roless ? (haha)

Reply:
BTW, it's OK for enums to be plural, because they're passed in as the enum name but are always dereferenced in use, eg. `IncorporatorTypes.PERSON`.

Reply:
😆 I agre. Change required here?

Reply:
Nah, you can do it in a future PR.

By the way, I can go-go-go, but we should work at a comfortable pace. No one needs to work evenings or weekends atm. Don't let me push you :)

Reply:
Sounds good to me!  😆 Don't worry about that one bit, no pressure in the world will outdo the pressure i get from inside the house!